src/ClusterBootstrap/services/job-insighter/job-insighter.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: DaemonSet

```

Comment:
If only need one replica, please use Deployment

Reply:
Changed to Deployment